README.md

```diff
@@ -19,15 +22,28 @@ In your .NET application, add the Eppo.Sdk Package from Nuget.
 dotnet add package Eppo.Sdk
 ```
 
+## Client Mode
+In order to have the most up-to-date _Assignment_ and _Bandit_ configuration, the SDK frequently polls the configuration server. This is not ideal in a client deployment such as a mobile or web app. For these client applications, this SDK can be run in **Client Mode** where configuration loading is done ad-hoc by calling the `EppoClient.RefreshConfiguration` method.
+
 ## Quick start
 
-Begin by initializing a singleton instance of Eppo's client. Once initialized, the client can be used to make assignments anywhere in your app.
+Begin by initializing a singleton instance of Eppo's client. Once initialized*, the client can be used to make assignments anywhere in your app.
+
+_*In **Client Mode**, `RefreshConfiguration` must be called before assignments can be made_
 
 #### Initialize once
 
 ```cs
 var eppoClientConfig = new EppoClientConfig('SDK-KEY-FROM-DASHBOARD');
+
+// For servers:
 var eppoClient = EppoClient.Init(eppoClientConfig);
+
+// For client (such as mobile) applications:
+var eppoClient = EppoClient.InitClientMode(eppoClientConfig);
```

Comment:
You must have considered it and have a good reason not to go down this path, but why not

```suggestion
var eppoClient = EppoClient.Init(eppoClientConfig, client_mode=true);
```

Reply:
More like agonized. I'm glad you asked this. I feel like I have 4/5 of a good reason here.
I do prefer the default parameter approach, but it may limit us in the future.

It's probably overkill, but I built deployment environment modes to be one of many instead of binary. If, in the future, we have some new deployment environment that is somehow neither client nor server, the `Init(config, clientMode)` method signature is no longer sufficient; changing that signature in a non-backward compatible way would require a major version bump (which may, in turn, hinder adoption). To be frank, I'm a little paranoid about changing the `Init` signature lest it prevent flexibility in the future.

Now, much of this is contrived and may be overly conservative/defensive. Considering this, what do you think? Would love to get your perspective.

Reply:
That's an excellent point, could perhaps do `mode=MODE.client` versus `mode=MODE.server` which allows for more options in the future but this clearly makes it most extensible